src/python/install.sh

```diff
@@ -344,13 +344,20 @@ install_python() {
         check_packages python3 python3-doc python3-pip python3-venv python3-dev python3-tk
         INSTALL_PATH="/usr"
 
-        ln -s "${INSTALL_PATH}/bin/python3" "${INSTALL_PATH}/bin/python"
-        ln -s "${INSTALL_PATH}/bin/pydoc3" "${INSTALL_PATH}/bin/pydoc"
-        ln -s "${INSTALL_PATH}/bin/python3-config" "${INSTALL_PATH}/bin/python-config"
-
-        # Add the current symlink but point it to "/usr" since python is at /usr/bin/python
-        mkdir -p "${PYTHON_INSTALL_PATH}"
-        add_symlink
+        local current_bin_path="${CURRENT_PATH}/bin"
+        if [ "${OVERRIDE_DEFAULT_VERSION}" = "true" ]; then
```

Comment:
`OVERRIDE_DEFAULT_VERSION` is useful for installing additional versions of python. 

Thanks for syncing it. However, looks like there is an existing bug for that.

Could we change this 👇  https://github.com/devcontainers/features/blob/74f75a51f25dfcb3734f6c43a8bee9a45867f024/src/python/install.sh#L343 

to 👇 

```
if [ ${version} = "os-provided" ] || [ ${version} = "system" ]; then 
```

Reply:
We wouldn't want to add all those versions to the path tho right?  As it is, the PYTHON_INSTALL_PATH is always updated, but the PATH is only updated if you aren't using a system version (which would already be in the PATH).  The $version is in a sub-folder from what I was reading.

Reply:
Oh wait - that link poiinted to the wrong spot.  Got it. Yeah, that was an existing bug.

Reply:
Great! thanks

Reply:
Fixed!